### PR TITLE
feat: add aio-lib-ims e2e tests

### LINKED ISSUE #19: Add missing e2e tests for `aio-lib-ims`
Tests for this SDK are missing and should be added.

### PR DESCRIPTION
**Pull request by shazron**
Closes #19

## How Has This Been Tested?

populate a `.env` with valid values
change repositories.json aio-lib-ims branch to `e2e-tests`
`node src/run.js`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: repositories.json [hunk 1]
```diff
@@ -42,5 +42,14 @@
...
     "doNotLog": [ "EVENTS_JWT_TOKEN" ]
+  },
+
+  "aio-lib-ims" : {
+    "repository": "https://github.com/adobe/aio-lib-ims",
+    "branch": "master",
+    "requiredEnv": [ "IMS_CLIENT_ID", "IMS_CLIENT_SECRET", "IMS_SIGNED_JWT" ],
+    "requiredAuth": "jwt",
+    "mapEnv": { "JWT_CLIENTID": "IMS_CLIENT_ID" , "JWT_CLIENT_SECRET": "IMS_CLIENT_SECRET", "JWT_SIGNED": "IMS_SIGNED_JWT" },
+    "doNotLog": [ "EVENTS_JWT_TOKEN" ]
```
[the remark this answers is not in the document]
Comment (shazron) [reply repositories.json:53]: oops. copypasta. will fix

[the remark this answers is not in the document]
Comment (shazron) [reply repositories.json]: fixed https://github.com/adobe/aio-e2e-tests/pull/32/commits/af09a5dc5ad9edab2653af5649513e3e710545f2

**Comment by shazron** [comment]
https://github.com/adobe/aio-lib-ims/pull/65 has been pulled in and released

File: repositories.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (moritzraho) [reply repositories.json]: nice !

**Comment by moritzraho** [comment]
LGTM but looks like tests fail on windows

**Comment by shazron** [comment]
> LGTM but looks like tests fail on windows

looks like the IMS tests pass now when I re-run the jobs, but the event tests are failing

**Comment by shazron** [comment]
aio-lib-ims e2e tests are passing but events e2e is not. Filed an issue for that: https://github.com/adobe/aio-e2e-tests/issues/36 and pulling this in.

**Pull request merged**